sdk/communication/communication-alpha-ids/samples-dev/getConfiguration.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Get the Alpha IDs configuration that's applied for the current resource
+ */
+
+import { AlphaIdConfiguration } from "@azure-tools/communication-alpha-ids";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export async function main() {
+  console.log("\n== Get the Alpha IDs configuration that's applied for the current resource ==\n");
+
+  // You will need to set this environment variable or edit the following values
+  const connectionString =
+    process.env.COMMUNICATION_SAMPLES_CONNECTION_STRING ||
+    "endpoint=https://resourceName.communication.azure.net/;accessKey=test-key";
+
+  // create new client
+  var config: AlphaIdConfiguration = {
```

Comment:
I am not seeing a new client being created here.

Reply:
That was my bad. I created this file very early on and forgot to update it once I mapped the API. It's fixed now.